Explorer/Assets/DCL/Infrastructure/ECS/Unity/ColorComponent/ColorDefaults.cs

```diff
@@ -8,6 +8,8 @@ public static class ColorDefaults
 
         public static readonly Color COLOR_BLACK = Color.black;
 
+        public static readonly Color COLOR_TRANSPARENT = new Color(0,0,0,0);
```

Comment:
you can use built-in constant `Color.clear`

Reply:
Ahh that's how it's called, I looked for it and couldn't believe Unity didn't already have it XD but didn't found it under `Color.transparent` XD. 

Thanks! addressed at https://github.com/decentraland/unity-explorer/pull/3646/commits/beb4b1c6008b120d52f9b0f50e60b9e1f3f29b17